build.gradle

```diff
@@ -98,17 +113,32 @@ project('ripc-transport-netty4') {
 
 		// Netty
 		compile "io.netty:netty-all:$nettyVersion"
+        compile "org.slf4j:slf4j-api:1.7.6"
 	}
 }
 
-project('ripc-composition-rxjava1') {
+project('ripc-rxjava1') {
 	description = 'Reactive IPC Composition Layer Implementation'
 	dependencies {
 		// ripc-tcp
-		compile project(":ripc-transport-netty4")
+		compile project(":ripc-protocol-tcp")
+        compile 'io.reactivex:rxjava:1.0.8'
+        compile 'io.reactivex:rxjava-reactive-streams:0.5.0'
+	}
+}
 
-		// RxJava 1.0
-		compile "io.reactivex:rxjava:$rxjava1Version"
+project('ripc-rxjava1-examples') {
+	description = 'Reactive IPC Composition Layer examples'
+
+    compileTestJava {
```

Comment:
Sure. I did not change the build file much from the version currently in the master. Let me look at it again to see what else should be modified/added.